web/src/components/notification-banner/notification-banner.tsx

```diff
@@ -42,28 +52,27 @@ export default function NotificationBanner({
   const [show, setShow] = useState(false);
 
   const el = useRef(null);
+  const { storageKey = '' } =
+    notification.kind == 'banner' && notification.bannerProps;
```

Comment:
Destructuring here is a bit weird, because you might end up checking for `false.storageKey`. I think:

```js
  const storageKey = notification.kind === 'banner' ? notification.bannerProps : '';
```

could work instead?